tests/test_metamod.py

```diff
@@ -1,6 +1,70 @@
 import os
+import subprocess
 from pathlib import Path
 
+import pytest
+from imod.couplers.metamod import MetaMod
+from imod.mf6 import Modflow6Simulation, open_cbc, open_hds
+from imod.msw import MetaSwapModel
+from numpy.testing import assert_array_almost_equal
+
+
+@pytest.fixture
+def metamod_model_sprinkling(
+    coupled_mf6_model: Modflow6Simulation,
+    msw_model: MetaSwapModel,
+    metaswap_lookup_table: Path,
+) -> MetaMod:
+
+    # Override unsat_svat_path with path from .env
+    msw_model.simulation_settings[
+        "unsa_svat_path"
+    ] = msw_model._render_unsaturated_database_path(metaswap_lookup_table)
+
+    return MetaMod(
+        msw_model,
+        coupled_mf6_model,
+        mf6_rch_pkgkey="rch_msw",
+        mf6_wel_pkgkey="wells_msw",
+    )
+
+
+@pytest.fixture
+def tmp_path_dev(
+    tmp_path: Path,
+) -> Path:
+    return tmp_path / "develop"
+
+
+@pytest.fixture
+def tmp_path_reg(
+    tmp_path: Path,
+) -> Path:
+    return tmp_path / "regression"
+
+
+def read_log_for_succes(logfile_path):
+    with open(logfile_path, "r") as logfile:
+        has_mf6_succes_message = False
+        has_msw_succes_message = False
+        for line in logfile:
+            if "Normal termination of simulation." in line:
+                has_mf6_succes_message = True
+            if "E N D   O F   M E T A S W A P" in line:
+                has_msw_succes_message = True
+
+    return has_mf6_succes_message, has_msw_succes_message
+
+
+def run_model(path: Path, imod_coupler_exec: Path):
+    logfile_path = path / "metamod.log"
+    with open(logfile_path, "w") as logfile:
+        subprocess.run(
+            [str(imod_coupler_exec), path / "metamod.toml"],
```

Comment:
```suggestion
            [imod_coupler_exec, path / "imod_coupler.toml"],
```

Reply:
Changed the name to ``imod_coupler.toml``